src/binary/coreir.cpp

```diff
@@ -106,17 +155,23 @@ int main(int argc, char *argv[]) {
   }
   
   if (options.count("l")) {
-    vector<string> files = splitString<vector<string>>(options["l"].as<string>(),',');
-    for (auto file : files) {
-      vector<string> f1parse = splitString<vector<string>>(file,'/');
-      string libfile = f1parse[f1parse.size()-1];
-      vector<string> f2parse = splitRef(libfile);
-      ASSERT(f2parse[1]=="so" || f2parse[1]=="dylib","Bad file: " + file);
-      string libname = f2parse[0].substr(10,f2parse[0].length()-10);
-
-      ASSERT(openLibHandles.count(file)==0,"Cannot REopen " + file);
-      void* libHandle = dlopen(file.c_str(),RTLD_LAZY);
-      ASSERT(libHandle,"Cannot open file: " + file);
+    vector<string> lpaths = {"/usr/local/lib","/usr/lib"};
+    if (auto lpath = std::getenv("LD_LIBRARY_PATH")) {
+      lpaths.push_back(lpath);
+    }
+    if (auto lpath = std::getenv("DYLD_LIBRARY_PATH")) {
+      lpaths.push_back(lpath);
+    }
```

Comment:
Might want to add these paths to the docs so users know where to install libraries